clients/da-vinci-client/src/main/java/com/linkedin/davinci/store/rocksdb/RocksDBStoragePartition.java

```diff
@@ -835,7 +835,7 @@ public long getDuplicateKeyCountEstimate() {
   }
```

Comment:
In this function: `getDuplicateKeyCountEstimate`, let us check whether RocksDB is closed or not since we are not maintaining a separate lock for `keyStatistics` and we can reuse the `readCloseRWLock` here.